nomad/structs/structs.go

```diff
@@ -1153,11 +1154,52 @@ type Node struct {
 	// updated
 	StatusUpdatedAt int64
 
+	// NodeEvents is the most recent set of events generated for the node
```

Comment:
When you add a constant per @preetapan 's feedback here: https://github.com/hashicorp/nomad/pull/3945/files#r173500252

then mention that constant in this comment.  (Otherwise when reading this comment I'm curious "most recent events, like how many though?")